example/package.json

```diff
@@ -8,6 +8,7 @@
     "ios": "expo run:ios",
     "web": "expo start --web",
     "prebuild": "expo prebuild",
+    "build": "npx react-native build-ios",
```

Comment:
What does this package do differently than the `ios` script above (`expo run:ios`)?

Reply:
It just runs xcodebuild vs run:ios runs the app on simulator.